src/KM_Controls.pas

```diff
@@ -6727,20 +6817,20 @@ constructor TKMMasterControl.Create;
 begin
   inherited;
 
-  CtrlPaintCount := 0;
-  fCtrl      := nil;
-  fCtrlDown  := nil;
-  fCtrlFocus := nil;
-  fCtrlOver  := nil;
-  fCtrlUp    := nil;
+  CtrlPaintCount  := 0;
+  fMasterPanel    := nil;
+  fCtrlDown       := nil;
+  fCtrlFocus      := nil;
+  fCtrlOver       := nil;
+  fCtrlUp         := nil;

```

Comment:
Actually we don't need to set these to 0/nil in constructor. Now would be a good time to remove this.